app/src/main/AndroidManifest.xml

```diff
@@ -12,14 +12,15 @@
         android:theme="@style/AppTheme">
         <activity
             android:name=".MainActivity"
-            android:label="@string/main_menu"
             android:theme="@style/AppTheme.NoActionBar">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".findAlbum"
+            android:parentActivityName=".MainActivity"></activity>
```

Comment:
You should close the `activity` tag with `/>` instead of doing `></activity>` since you have nothing inside the activity element